src/components/overworld/objects/Tube.tsx

```diff
@@ -55,7 +55,7 @@ function Tube({
 		<>
 			<mesh name={name} position={position}>
 				<primitive object={tubeGeometry} />
-				<meshPhysicalMaterial roughness={0.05} transmission={1} thickness={0.5} side={DoubleSide} />
+				<meshPhysicalMaterial roughness={0.05} thickness={0.5} side={DoubleSide} />
```

Comment:
Where is transmission?

Reply:
forgot to put it back in meh

Reply:
One more time and you can stukc your meh in yo ass

Reply:
ok